.eslintrc.cjs

```diff
@@ -3,19 +3,27 @@ module.exports = {
     browser: true,
     es2021: true
   },
-  extends: ['airbnb', 'airbnb/hooks', 'plugin:react/recommended', 'plugin:prettier/recommended', 'plugin:storybook/recommended'],
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react/recommended',
+  ],
   overrides: [],
   parserOptions: {
     ecmaVersion: 'latest',
     sourceType: 'module'
   },
   plugins: ['react'],
   rules: {
-    'react/react-in-jsx-scope': 0,
-    'no-unused-vars': 'warn',
-    'no-undef': 'warn',
-    quotes: ['error', 'single', {
-      allowTemplateLiterals: true
-    }]
-  }
+    'linebreak-style': ['error', 'windows'],
```

Comment:
It's not recommended to set this rule to a specific platform because it can cause issues when sharing code across different platforms. Instead, you should set the linebreak-style rule to "off" to allow any line ending style in your code.